vector/src/main/java/im/vector/app/features/workers/signout/ServerBackupStatusViewModel.kt

```diff
@@ -118,7 +126,7 @@ class ServerBackupStatusViewModel @AssistedInject constructor(@Assisted initialS
                     if (
                             crossSigningInfo.getOrNull() == null
                             || (crossSigningInfo.getOrNull()?.isTrusted() == true
-                            && pInfo.getOrNull()?.allKnown().orFalse())
+                                    && pInfo.getOrNull()?.allKnown().orFalse())
```

Comment:
Can we revert this modification, in order to keep the Element code for this line?